apprentice_teacher.rb

```diff
@@ -26,15 +15,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
```

Comment:
I notice this method in two spots. How could it be refactored and abstracted out into the parent class?